src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionWarnOthersViewControllerTests.swift

```diff
@@ -22,16 +22,19 @@ import XCTest
 @testable import ENA
 
 class ExposureSubmissionWarnOthersViewControllerTests: XCTestCase {
-
-	override func setUp() {
-		super.setUp()
+	
```

Comment:
Use`MockTestStore` and add linebreaks after parameters in this file as well

Reply:
That is a good point. The code in fact was already previous code. I have mocked the store now.

Reply:
Done